ethereum/executionlayer/src/main/java/tech/pegasys/teku/ethereum/executionlayer/client/auth/JwtSecretKeyLoader.java

```diff
@@ -71,7 +72,7 @@ private void writeGeneratedKeyToFile(final Key key) {
   private SecretKeySpec loadSecretFromFile(final String jwtSecretFile) {
     final Path filePath = Paths.get(jwtSecretFile);
     try {
-      final Bytes bytesFromHex = Bytes.fromHexString(Files.readString(filePath));
+      final Bytes bytesFromHex = Bytes.fromHexString(Files.readAllLines(filePath).get(0));
```

Comment:
I'd probably just use `Field.readString(filePath).trim()` here so that we discard newline and any trailing spaces etc.

Reply:
Yeah. I've put dirty code in this PR:)
Tomorrow I'll cleanup!